server/server.js

```diff
@@ -15,6 +15,7 @@ var uploader = require('./uploader.js');
 var bed = require('./bed.js');
 var mapping = require('./mapping.js');
 var log = require('./log.js');
+var datadb = require('./datadb.js');
```

Comment:
if the file names are two words, usually by js convention it's concatenated by dash, i.e. data-db.js. The namespace needs to be camelCase however.